spec/index.html

```diff
@@ -83,14 +121,19 @@
     @media (max-width: 767px) {
       table { word-break: break-all; }
       .separated thead tr th { border:1px solid black; padding: .2em; min-width: 10vw }
+      .separated tbody tr td:first-child,
+      .separated tbody tr th:first-child {max-width: 220px; overflow-wrap: anywhere;}
       .separated thead tr th:nth-child(2) { word-break: break-all; }
       .separated tbody tr td:nth-child(2) {padding: 3px 2px;}
       .separated tbody tr td:nth-child(3) {padding: 3px 2px;}
       .separated tbody tr td { border:1px solid black; text-align: center; min-width: 10vw }
+
+      body.darkmode .separated thead tr th { border:1px solid white; }
+      body.darkmode .separated tbody tr td { border:1px solid white; }
```

Comment:
If we are lazy I think we can just write `.separated tbody tr td { border:1px solid; }`. The default border color is black in light mode and white in dark mode (same below)

Reply:
Good suggestions, it should be simpler and still look good, now.